activerecord/CHANGELOG.md

```diff
@@ -469,6 +469,18 @@
     end
     ```
 
+    Active Record logs will also include timing info for the duration of how long
+    the async query waited for the mutex lock. This timing is useful to know how
+    effecient or not the query is.
+
+    ```
+    DEBUG -- :   Category Load (62.1ms)  SELECT * FROM `categories` LIMIT 50
+    DEBUG -- :   ASYNC Post Load (64ms) (db time 126.1ms)  SELECT * FROM `posts` LIMIT 100
+    ```
+
+    The duration in the first set of parens is the lock wait time, and the second set of parens
```

Comment:
> lock wait time

Again, I don't think "lock wait time" is clear enough for people not aware of the implementation.

So maybe something:

> How long the main thread was blocked waiting for the results.